README.md

```diff
@@ -60,8 +60,9 @@ oc login -u kubeadmin -p 12345678 https://api.crc.testing:6443
 ```
 
 * attach libvirt default network to the crc (default IP 192.168.122.10). This network is used as a vlan trunk to isolate the networks using vlans.
+Try running `crc ip` command to get the default crc network ip.
 ```bash
-make crc_attach_default_interface
+CRC_DEFAULT_NETWORK_IP=$(crc ip) make crc_attach_default_interface
```

Comment:
I don't understand this patch. $(crc ip) is going to return the IP for the crc vm on the crc network (192.168.130.xx). The only place $CRC_DEFAULT_NETWORK_IP is used is to add add an dhcp reservation on the default network (192.168.122.xx).

Reply:
I agree.  It seems like the wrong IP address is being used here.

Reply:
Thank you @slagle and @abays for reviewing it. I misunderstood this step earlier  now it is clear.
crc vm get's default ip 192.168.130.11, and with crc_attach_default_interface we want to also add 192.168.122.10 ip to crc vm so other libvirt vms can access it. I will close this pr.